README.md

```diff
@@ -68,117 +99,85 @@ nslookup ${He_Name}.azurecr.io
 
 Create Resource Groups
 
-- When experimenting with this app, you should create new resource groups to avoid accidentally deleting resources
-
-  - If you use an existing resource group, please make sure to apply resource locks to avoid accidentally deleting resources
+> When experimenting with this app, you should create new resource groups to avoid accidentally deleting resources
+>
+> If you use an existing resource group, please make sure to apply resource locks to avoid accidentally deleting resources
 
 - You will create 3 resource groups
   - One for ACR
   - One for App Service or AKS, Key Vault and Azure Monitor
-  - One for Cosmos DB (see create and load sample data to Cosmos DB step)
+  - One for Cosmos DB
 
 ```bash
 
 # set location
-export He_Location=centralus
+export He_Location=
```

Comment:
Should we show an example location (for format)? Not sure on what level of Azure experience we should assume

Reply:
changed back to centralus